slsim/Halos/halos_lens.py

```diff
@@ -0,0 +1,1896 @@
+from lenstronomy.Util.util import make_grid
+import numpy as np
+import astropy.units as u
+from lenstronomy.LensModel.lens_model import LensModel
+from lenstronomy.Cosmo.lens_cosmo import LensCosmo
+import warnings
+import math
+import time
+import multiprocessing
+from collections.abc import Iterable
+
+
+def deg2_to_cone_angle(solid_angle_deg2):
```

Comment:
this could be moved to an utility file as it might be useful for other routines as well